lib/launcher.ts

```diff
@@ -176,7 +177,18 @@ let initFn = function(configFile: string, additionalConfig: Config) {
         // 4) Run tests.
         let scheduler = new TaskScheduler(config);
 
+        process.on('uncaughtException', (e: Error) => {
+          let errorCode = ErrorHandler.parseError(e);
+          if (errorCode) {
+            logger.error(e.stack);
+            process.exit(errorCode);
+          } else {
+            logger.error(e.stack);
+          }
+        });
+
         process.on('exit', (code: number) => {
+          console.log(code);
```

Comment:
Should we be logging this? If someone wants to use it, they can just use the exit code.